src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -2010,113 +2038,124 @@ msgstr "Bestand"
 msgstr "Beslissing op zijn laatst:"
 
-#: open_inwoner/cms/cases/views/status.py:386
-#: open_inwoner/cms/cases/views/status.py:559
-#: open_inwoner/openzaak/api_models.py:92
-#: open_inwoner/openzaak/notifications.py:167
+#: open_inwoner/cms/cases/views/status.py:399
+#: open_inwoner/cms/cases/views/status.py:583
+#: open_inwoner/openzaak/api_models.py:90
+#: open_inwoner/openzaak/notifications.py:175
+#: open_inwoner/openzaak/tests/test_api_models.py:92
 #: open_inwoner/userfeed/hooks/case_status.py:28
-#: open_inwoner/userfeed/hooks/case_status.py:94
+#: open_inwoner/userfeed/hooks/case_status.py:103
 msgid "No data available"
 msgstr "Geen data beschikbaar"
 
-#: open_inwoner/cms/cases/views/status.py:411
-#: open_inwoner/cms/cases/views/status.py:587
+#: open_inwoner/cms/cases/views/status.py:424
+#: open_inwoner/cms/cases/views/status.py:611
 msgid "Bekijk aanvraag"
 msgstr "Bekijk aanvraag"
 
-#: open_inwoner/cms/cases/views/status.py:655
+#: open_inwoner/cms/cases/views/status.py:680
 msgid "Gegevens"
 msgstr "Gegevens"
 
-#: open_inwoner/cms/cases/views/status.py:661
+#: open_inwoner/cms/cases/views/status.py:686
 #: open_inwoner/templates/pages/cases/status_inner.html:26
 msgid "Documenten"
 msgstr "Documenten"
 
-#: open_inwoner/cms/cases/views/status.py:701
-#: open_inwoner/openzaak/tests/test_documents.py:233
+#: open_inwoner/cms/cases/views/status.py:731
+#: open_inwoner/openzaak/tests/test_documents.py:236
 #, python-brace-format
 msgid "Document van zaak gedownload {case}: {filename}"
 msgstr "Document van zaak gedownload {case}: {filename}"
 
-#: open_inwoner/cms/cases/views/status.py:736
+#: open_inwoner/cms/cases/views/status.py:766
 #, python-brace-format
 msgid "An error occured while uploading file {filename}"
 msgstr "Een fout is opgetreden bij het uploaden van {filename}"
 
-#: open_inwoner/cms/cases/views/status.py:780
+#: open_inwoner/cms/cases/views/status.py:816
 #, python-brace-format
 msgid "Document was uploaded for {case}: {filename}"
 msgstr "Document {filename} is geüpload voor {case}"
 
-#: open_inwoner/cms/cases/views/status.py:788
+#: open_inwoner/cms/cases/views/status.py:824
 #, python-brace-format
 msgid "Wij hebben **{num_uploaded} bestand(en)** succesvol geüpload:"
 msgstr "Wij hebben **{num_uploaded} bestand(en)** succesvol geüpload:"
 
-#: open_inwoner/cms/cases/views/status.py:854
-#: open_inwoner/cms/cases/views/status.py:886
+#: open_inwoner/cms/cases/views/status.py:900
+#: open_inwoner/cms/cases/views/status.py:938
 #, python-brace-format
 msgid "Case: {case_identification}"
 msgstr "Zaak: {case_identification}"
 
-#: open_inwoner/cms/cases/views/status.py:879
+#: open_inwoner/cms/cases/views/status.py:931
 #: open_inwoner/openklant/views/contactform.py:77
 msgid "Probleem bij versturen van de vraag."
 msgstr "Probleem bij versturen van de vraag."
 
+#: open_inwoner/cms/cases/views/status.py:1079
+msgid ""
+"The link you clicked on has expired. Please find your case in the list below."
+msgstr ""
```

Comment:
"De link die u gebruikte is verlopen. Uw aanvraag is via onderstaand overzicht te vinden."